Applications/spire/source/charting/chart_view.cpp

```diff
@@ -12,6 +11,13 @@ using namespace boost::posix_time;
 using namespace Nexus;
 using namespace Spire;
 
+namespace {
+  QVariant chart_value_to_variant(ChartValue::Type type,
```

Comment:
When naming a function, try to leverage the fact that functions can be overloaded, this means that the function's parameters are a part of its signature. You can rename the function to ```to_variant``` and leverage the fact that ```ChartValue``` is one of the function parameters. This naming convention has benefits when writing generic code.